lyse/__init__.py

```diff
@@ -703,12 +705,21 @@ def globals_groups(self):
             except KeyError:
                 return []   
                 
-    def globals_diff(self, other_run, group=None):
-        return globals_diff(self, other_run, group)            
+    def globals_diff(self, other_shot, group=None):
+        return globals_diff(self, other_shot, group)            
     
-        
-class Sequence(Run):
-    def __init__(self, h5_path, run_paths, no_write=False):
+
+class Run(Shot):
+    def __init__(self, *args, **kwargs):
+        warnings.warn("The 'Run' class has been renamed to 'Shot' (but is otherwise compatible). 'Run' will be removed in lyse v4.0. Please update your analysis code to use the 'Shot` class", DeprecationWarning)
+        super().__init__(*args, **kwargs)
+
+    def globals_diff(self, other_run, group=None):
+        return globals_diff(self, other_run, group)  
```

Comment:
I think the inherited version of this method would work fine right? Is this included just in case users had explicitly called this method with `other_run=...`, using it as a named argument rather than a positional one?

Reply:
Yeah, that was the reasoning. Unlikely (and I argued above unnecessary) but the opportunity to do it was available and it's more likely that someone is using this method than the underlying `globals_diff` function.